test/4.public.UrlChecker.js

```diff
@@ -139,9 +139,83 @@ describe("PUBLIC -- UrlChecker", function()
 			}).enqueue( conn.absoluteUrl );
 		});
 	});
-	
-	
-	
+
+
+
+	describe("caching", function()
+	{
+		var options;
+		beforeEach(function()
+		{
+			options = utils.options();
```

Comment:
I do too, but I felt that these are minor enough to not be a performance issue. Though, more importantly--to me, at least--it helps keep the tests more self-contained and consistent with other tests that have different custom options.